docs-ref-services/sql.md

```diff
@@ -5,60 +5,90 @@ keywords: Azure, Python, SDK, API, SQL, database , pyodbc
 author: lisawong19  
 ms.author: liwong
 manager: douge
-ms.date: 06/09/2017
+ms.date: 07/11/2017
 ms.topic: article
 ms.prod: azure
 ms.technology: azure
-ms.devlang: java
-ms.service: 
+ms.devlang: python
+ms.service: sql-database
 ---
 
 # Azure SQL Database libraries for Python
 
 ## Overview
 
-Work with data stored in  [Azure SQL Database](https://docs.microsoft.com/azure/sql-database/sql-database-technical-overview) from Python with the Microsoft ODBC driver. 
+Work with data stored in  [Azure SQL Database](https://docs.microsoft.com/azure/sql-database/sql-database-technical-overview) from Python with the Microsoft ODBC driver and pyodbc. 
 
-The management libraries provide an interface to create, manage, and scale Azure SQL Database deployments from your Python code. Set up and manage databases in [elastic pools](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-elastic-pool) to share resources and configure databases across multiple regions from your code.
+## Client ODBC driver and pyodbc
+Connect to the Azure SQL Database from your applications using the [Microsoft ODBC driver and pyodbc](https://docs.microsoft.com/azure/sql-database/sql-database-connect-query-python#install-the-python-and-database-communication-libraries).
```

Comment:
I would say "pip install pyodbc". For more detail, click here "install-the-python-and-database-communication-libraries"

Since in 99% of the scenarios, "pip install "is enough.